README.md

```diff
@@ -70,6 +70,21 @@ The following attributes are available for `oak-d` cameras:
 Although not a config attribute, you can also configure the module to output debug logs.
 This is done by using the `-debug` flag when starting the Viam server in order for module debug logs to be piped through to stdout e.g. `viam-server -debug -config path/to/your/config.json`.
 
+
+### Set udev rules on Linux
+
+- Failed to boot the device: 1.3-ma2480, err code 3
+- Failed to find device (ma2480), error message: X_LINK_DEVICE_NOT_FOUND
+- [warning] skipping X_LINK_UNBOOTED device having name "<error>"
+- Insufficient permissions to communicate with X_LINK_UNBOOTED device with name "1.1". Make sure udev rules are set
+
+If you see any of the above errors, you may need to set udev rules on your Linux machine. See [here](https://docs.luxonis.com/projects/api/en/latest/pages/udev_rules/) for more information.
```

Comment:
[This](https://docs.luxonis.com/projects/api/en/latest/pages/udev_rules/) link isn't working for me. Could you double check it?

Reply:
Thanks! Fixed link.